src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -159,7 +159,7 @@ export class ServiceScene extends SceneObjectBase<ServiceSceneState> {
     >
   ) {
     super({
-      $data: getServiceSceneQueryRunner(),
+      $data: undefined,
```

Comment:
A tad confused as to why?

Reply:
Definitely. Based on https://github.com/grafana/logs-drilldown/blob/main/src/Components/ServiceScene/ActionBarScene.tsx#L35 and the fact that the data source itself can have a custom or default line limit, combined with `getMaxLines` having to generate an "exploration prefix" for the stored value, `getMaxLines` receives a sceneRef which doesn't exist on the constructor.

https://github.com/grafana/logs-drilldown/blob/feb61e3f363d6883c0cb9cc08d8535e4eb3a4af5/src/services/store.ts#L216

So, to create a query runner for logs, it needs to read `maxLines`, and have a sceneRef to pass around.